frontend/conf/riff-raff.yaml

```diff
@@ -8,6 +8,9 @@ deployments:
       amiTags:
         Recipe: xenial-membership
         AmigoStage: PROD
+      cloudFormationStackName: "Memb"
+      prependStackToCloudFormationStackName : false
+      appendStageToCloudFormationStackName : true
```

Comment:
Without these three lines, we get:

![image](https://cloud.githubusercontent.com/assets/52038/25943821/16e79528-3639-11e7-9f6e-35c737020cc1.png)